Admin.js

```diff
@@ -11,7 +11,7 @@
 
 var Admin = {
 
-    collectionCounters: [],
+    lolCounters: [],
 
     /**
```

Comment:
- eslint: Missing JSDoc @returns for function.
- eslint: Missing JSDoc parameter type for 'subject'.
- eslint: Missing JSDoc parameter description for 'subject'.